parts/k8s/addons/kubernetesmasteraddons-pod-security-policy.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   annotations:
     seccomp.security.alpha.kubernetes.io/allowedProfileNames: "*"
   labels:
-    addonmanager.kubernetes.io/mode: Reconcile
+    addonmanager.kubernetes.io/mode: EnsureExists
```

Comment:
After further testing did you determine that `EnsureExists` is a better option in practice for psp?

Reply:
Yes. The original behavior on the old func https://github.com/Azure/acs-engine/pull/2048/files#diff-f2f5404d19a8ead71fce9bc62331f682R616 only create the PSPs once - without reconciliating them in case of updates/deletes - which makes sense. With `EnsureExists`, the addon manager will create only once (instead of `Reconcile` option)